pages/Configuring/Multi-GPU.md

```diff
@@ -0,0 +1,132 @@
+# Table of contents
+
+{{< toc >}}
+
+# General
+
+If your host machine uses multiple GPUs, you may want to primarily use a GPU for rendering all the elements for Hyprland including windows, animations, hardware acceleration for certain applications, etc.
+
+This setup is very common in the likes of gaming laptops, GPU-passthrough(without VFIO) capable hosts, and if you have multiple GPUs in general.
+
+# Detecting GPUs
+
+For this case, the writer is taking the example of their laptop. Here are some of the details for the same
+
+Upon running `lspci | grep VGA`, I can list all the video devices installed on this laptop
```

Comment:
In the case of gaming laptops, sometimes the discrete GPU shows up as 3D controller instead of VGA controller.
```suggestion
Upon running `lspci | grep -E 'VGA|3D'`, I can list all the video devices installed on this laptop
```